### PR TITLE
Bump to cardano-node 1.29.0-rc1

### PR DESCRIPTION
**Pull request by Anviking**
- [x] Bump to cardano-node 1.29.0-rc1
- [x] Bump to the actual tag, when it exists
- <s>Move snapshot to cardano-haskell</s>
- [ ] Update readme
- [x] Fix arcane windows hydra issues - might not have been real errors

### Issue Number

ADP-1081

### CODE REVIEW

File: stack.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 #
 # NOTE: Remember to update the version matrix in README.md when
 # bumping the Cardano version.
-resolver: https://raw.githubusercontent.com/input-output-hk/cardano-haskell/c2aa1078ca8261d98cfce16d1131fbc687f6692b/snapshots/cardano-alonzo-purple-1.0.1.yaml
+resolver: tmp-snapshot.yaml
```
Comment (rvl) [inline stack.yaml]: If you would like to move the snapshot back into `stack.yaml`, feel free.

**Review by rvl — COMMENTED** [review]
Thanks

File: stack.yaml (hunk 1 above)
Comment (Anviking) [reply stack.yaml]: I'm not sure if it helps the cache issues, but very happy to try. And even if not, it's easier to bump `cabal.project` and `stack.yaml` in the same repo than having the cardano-haskell indirection.

Comment (Anviking) [reply stack.yaml]: Yup, the buildkite cache does still break 😕

Comment (rvl) [reply stack.yaml]: OK, pity - sounds like we need add a cache key then, based on the content of `stack.yaml`.

**Comment by Anviking** [comment]
bors try

**Comment by Anviking** [comment]
bors try

**Review by rvl — APPROVED** [review]
Thanks :+1:

Please do squash the fixup commits before merging.

**Comment by rvl** [comment]
Those Hydra build failures seem to have been due to some builds getting killed by a sysadmin. Unfortunately the killed builds (and builds depending on them) were cached as failed, and so kept on failing until I restarted them all.

**Comment by Anviking** [comment]
I thought I tried restarting them before, but ran into exactly the same issues 🤔 — but I might have misread it

**Comment by Anviking** [comment]
Thanks for the help @rvl !

bors r+

**Comment by Anviking** [comment]
No, the recent failure is recent, and not cached.

Previously I saw:
```
---> killing remote-iserv...
installing
Installing library in /nix/store/v1dcwk9jk4y3bb7g6rrh9cnxv9b59a7z-cardano-wallet-core-lib-cardano-wallet-core-x86_64-w64-mingw32-2021.8.11/lib/x86_64-windows-ghc-8.10.5/cardano-wallet-core-2021.8.11-5hdA5nlgvpLBtT3ZFL9INo
post-installation fixup
patching script interpreter paths in /nix/store/v1dcwk9jk4y3bb7g6rrh9cnxv9b59a7z-cardano-wallet-core-lib-cardano-wallet-core-x86_64-w64-mingw32-2021.8.11
```
which I thought was related to https://input-output-rnd.slack.com/archives/CAP8NM7N0/p1629473846081300?thread_ts=1623935006.211500&cid=CAP8NM7N0

But this time it just failed with
```
Progress.o, dist/build/Cardano/Wallet/Primitive/SyncProgress.dyn_o )
[59 of 93] Compiling Cardano.Wallet.Network ( src/Cardano/Wallet/Network.hs, dist/build/Cardano/Wallet/Network.o, dist/build/Cardano/Wallet/Network.dyn_o )
[60 of 93] Compiling Cardano.Wallet.DB.Sqlite.Types ( src/Cardano/Wallet/DB/Sqlite/Types.hs, dist/build/Cardano/Wallet/DB/Sqlite/Types.o, dist/build/Cardano/Wallet/DB/Sqlite/Types.dyn_o )
[61 of 93] Compiling Cardano.Pool.DB.Sqlite.TH ( src/Cardano/Pool/DB/Sqlite/TH.hs, dist/build/Cardano/Pool/DB/Sqlite/TH.o, dist/build/Cardano/Pool/DB/Sqlite/TH.dyn_o )
/nix/store/qg8qhrxiab3r87xmaxbq565g1g8bnl57-stdenv-linux/setup: line 1311:   119 Killed                  $SETUP_HS build lib:cardano-wallet-core -j$(($NIX_BUILD_CORES > 4 ? 4 : $NIX_BUILD_CORES))
builder for '/nix/store/rkfgswz6rg31qy9kqgyykf5h0qf6hd8c-cardano-wallet-core-lib-cardano-wallet-core-2021.8.11.drv' failed with exit code 137
```

**Pull request merged**